packages/accounts-controller/CHANGELOG.md

```diff
@@ -8,8 +8,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.1.0]
 ### Changed
-- Bump `@metamask/auto-changelog` to `^3.4.2` ([#1905](https://github.com/MetaMask/core/pull/1905))
-- Bump @metamask/auto-changelog from 3.2.0 to 3.4.0 ([#1870](https://github.com/MetaMask/core/pull/1870))
 - Update the `onKeyringStateChange` and `onSnapStateChange` methods, and remove the `keyringApiEnabled` on AccountsController ([#1839](https://github.com/MetaMask/core/pull/1839))
```

Comment:
This feature was supposed to be used prior to the account snap api going live on main, but since the feature was merged before being integrated into the extension i removed the flag.

Reply:
We can change this to a 4.0.0